### ISSUE TITLE
Unable to open dashboard login page [error] crasher: initial call: application_master:init/4, pid: <0.442.0>, registered_name: [], exit: {{bad_return,{{emqx_conf_app,start,[normal,[]]}

### ISSUE THREAD

**Issue by kedar-hatkar** [emqx/emqx#12090]
### What happened?

error log observed hence unable to test 5.3.2 version locally

### What did you expect to happen?

Run EMQX 5.3.2 broker normally.

### How can we reproduce it (as minimally and precisely as possible)?

1) Download EMQX 5.3.2 on windows. Keep default configuration in emqx.conf
2) Open command prompt run as administrator and emqx install. Service gets added in service explorer
3) Start emqx service from service explorer
4) error log seen in log file

### Anything else we need to know?

attaching debug logs and config files. in Service explorer service status was showing as running

In case of EMQX 5.3.1, no error observed and functionality was working properly

### EMQX version

sysdescr  : EMQX
version   : 5.3.2

### OS version

BuildNumber  Caption                   OSArchitecture  Version

19045        Microsoft Windows 10 Pro  64-bit          10.0.19045

### Log files

[app.2023.12.03.18.05.17.config.txt](https://github.com/emqx/emqx/files/13543009/app.2023.12.03.18.05.17.config.txt)
[vm.2023.12.03.18.05.17.args.txt](https://github.com/emqx/emqx/files/13543011/vm.2023.12.03.18.05.17.args.txt)
[emqx.conf.txt](https://github.com/emqx/emqx/files/13543012/emqx.conf.txt)
[emqx.log.1.txt](https://github.com/emqx/emqx/files/13543013/emqx.log.1.txt)

**Comment by lafirest** [comment]
@kedar-hatkar 
Thank for your feedback, would you please consider using the Docker? We are plan to retire the windows version.

**Comment by kedar-hatkar** [comment]
Hi EMQX team, please don't retire windows version very soon. Lot of small scale app are using EMQX on windows with basic configuration.

I am waiting for feature like Session persistence with RocksDB  and OTP 26 integration in windows version. I doubt these are available in 5.3.2 version currently

**Comment by keynslug** [comment]
Hi @kedar-hatkar,

This suggests that there was actually some problem with the logging configuration.
```
2023-12-03T18:05:51.682000+05:30 [error] crasher: initial call: application_master:init/4, pid: <0.442.0>, registered_name: [], exit: {{bad_return,{{emqx_conf_app,start,[normal,[]]},{'EXIT',{{case_clause,undefined},[{emqx_config_logger,tr_console_handler,1,[{file,"emqx_config_logger.erl"},{line,131}]},{emqx_config_logger,tr_handlers,1,[{file,"emqx_config_logger.erl"},{line,126}]},{emqx_config_logger,do_refresh_config,1,[{file,"emqx_config_logger.erl"},{line,50}]},{emqx_conf_app,start,2,[{file,"emqx_conf_app.erl"},{line,38}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]}, ancestors: [<0.441.0>], message_queue_len: 1, messages: [{'EXIT',<0.443.0>,normal}], links: [<0.441.0>,<0.44.0>], dictionary: [], trap_exit: true, status: running, heap_size: 376, stack_size: 28, reductions: 169; neighbours:
```

I haven't been able to reproduce the crash with the config you've provided, so the only thing I can recommend now is to: 1. Clean everything in the data directory.
2. Start with the default config, or update the logging config to a recommended form:
```
log.file.default {
  enable = true
  level = debug
  file = "log/emqx.log"
  max_size = "10MB"
}
```

**Comment by sbussy** [comment]
Hi,

I have tried to install version from 5.2.1 to 5.3.2 on a VM running on Windows Server 2012 R2 Standard and having still the same issue as @kedar-hatkar as windows service. Everything running fine on v5.3.2 with CLI "emqx start" but I need it running as a windows service.

Here is the log I have when running emqx_5.3.2 windows service :
`2023-12-14T11:35:22.043000+01:00 [error] crasher: initial call: application_master:init/4, pid: <0.645.0>, registered_name: [], exit: {{bad_return,{{emqx_conf_app,start,[normal,[]]},{'EXIT',{{case_clause,undefined},[{emqx_config_logger,tr_console_handler,1,[{file,"emqx_config_logger.erl"},{line,129}]},{emqx_config_logger,tr_handlers,1,[{file,"emqx_config_logger.erl"},{line,124}]},{emqx_config_logger,do_refresh_config,1,[{file,"emqx_config_logger.erl"},{line,48}]},{emqx_conf_app,start,2,[{file,"emqx_conf_app.erl"},{line,40}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]}, ancestors: [<0.644.0>], message_queue_len: 1, messages: [{'EXIT',<0.646.0>,normal}], links: [<0.644.0>,<0.44.0>], dictionary: [], trap_exit: true, status: running, heap_size: 610, stack_size: 28, reductions: 190; neighbours:
`

**Comment by qzhuyan** [comment]
> Hi EMQX team, please don't retire windows version very soon. Lot of small scale app are using EMQX on windows with basic configuration.
> 
> I am waiting for feature like Session persistence with RocksDB and OTP 26 integration in windows version. I doubt these are available in 5.3.2 version currently

could you run VM in windows?

**Comment by id** [comment]
Sorry folks, we have stopped supported native Windows deployments.
